changelog.d/13813.bugfix

```diff
@@ -1 +1 @@
-Fix a bug where redactions were not being sent over federation if we did not have the original event.
+Fix a bug where redactions were not being sent over federation if we did not have the original event (#13813).
```

Comment:
I think the former was fine---towncrier will append the PR number when we build the release notes.

Reply:
Ah cool I didn't know that. I saw pr numbers on commits when I was chasing down that memory leak and found it really helpful and I wanted to make sure I was doing the helpful thing.